### ISSUE TITLE
Please support ARM64 extensions

### ISSUE THREAD

**Issue by TimPosey2** [usysware/dpack#49]
[Visual Studio 2022 now supports ARM64 extensions](https://devblogs.microsoft.com/visualstudio/now-introducing-arm64-support-for-vs-extensions/).  Would you mind updating the VSIX manifest with arm64 support?

**Comment by sergey-visual-studio** [comment]
Updating manifest is easy. What that will lead to is need for more testing, which I'm not equipped to handle personal time and environment wise. And there is also Azure to modify to post new build along side of the existing one. Doable yes, it'll just be on a back burner for now.

**Comment by KenDFish** [comment]
I've been desperate for a working numbered-bookmark package that's compatible with ARM64, but couldn't find anything installable from the VS Marketplace. However, I've grabbed the DPack source, edited the source.extension.vsixmanifest to include: 

![image](https://github.com/sergey-visual-studio/dpack/assets/6280435/f2797b62-3ab4-4bea-b7ef-a97bd5ae5649)

.. and built and installed a release .VSIX. It's appears to be working fine - early days yet, but numbered bookmarks work perfectly. I can ditch Resharper now - thank you so much!

**Comment by sergey-visual-studio** [comment]
Yep, that, vsixmanifest that is, is what needs to be modified. The challenges it creates are related to build and distribution, ie building 2 separate versions, modifying the pipeline to upload both versions onto VS Marketplace. That's basically why it hasn't happened yet.
